docs/target-browser/README.md

```diff
@@ -8,8 +8,9 @@ Plugin currently supports GET operation on target-browser's `/backupplan`, `/bac
   
   1. krew - kubectl-plugin manager. Install from [here](https://krew.sigs.k8s.io/docs/user-guide/setup/install/)
   2. kubectl - kubernetes command-line tool. Install from [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-  3. TVK's web-backend service and target-browser server should be up and running.
-  
+  3. Target CR should have `browsingEnabled` field set to `true` in status[JSON Path - `target.status.browsingEnabled`].
```

Comment:
I think we also need a way for UI users to figure out if they have browsing enabled on their target
@seema Can you help over here